pkg/kubelet/status/status_manager.go

```diff
@@ -155,7 +155,13 @@ func (m *manager) Start() {
 func (m *manager) GetPodStatus(uid types.UID) (v1.PodStatus, bool) {
 	m.podStatusesLock.RLock()
 	defer m.podStatusesLock.RUnlock()
-	status, ok := m.podStatuses[m.podManager.TranslatePodUID(uid)]
+	// Func removeOrphanedPodStatuses in kubelet_pods.go calls
+	// func RemoveOrphanedStatuses in status_manager.go with both
+	// static pod UIDs and mirror pod UIDs, which in turn
+	// iterates over the podStatuses map of this manager struct
+	// that is then forced to hold types.UID keys.
+	// Thus, we resolve and type cast back again.
```

Comment:
I don't totally understand this comment. Is this just explaining why the podStatuses map needs to key off of a types.UID type?